op-program/host/host.go

```diff
@@ -21,10 +22,13 @@ import (
 	oppio "github.com/ethereum-optimism/optimism/op-program/io"
 	"github.com/ethereum-optimism/optimism/op-program/preimage"
 	opservice "github.com/ethereum-optimism/optimism/op-service"
+	opclient "github.com/ethereum-optimism/optimism/op-service/client"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/log"
 )
 
+const maxAttempts = math.MaxInt // Succeed or die trying
```

Comment:
I'd probably rename this to `maxRPCRetries` since it doesn't have the context of being in a file dedicated to retrying to stuff anymore.